website/src/e-commerce/README.md

```diff
@@ -0,0 +1,21 @@
+# E-commerce demo
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/create-instantsearch-app) by [Algolia](https://algolia.com)._
+
+## Get started
```

Comment:
Shouldn't we mention production build here too? 🤔

Reply:
You mean explaining the build script? I'm open to suggestions for the README because I'd like to make it more exhaustive.